common/culture/cultures/goblin.txt

```diff
@@ -2,12 +2,11 @@
 	graphical_cultures = {
```

Comment:
It's just black for me.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/115928287-eccf7280-a496-11eb-8e4f-d3c133711426.png)

</details>